packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -47,6 +49,34 @@ class DescopeWc extends BaseDescopeWc {
 
   flowState: State<FlowState>;
 
+  log = {
+    error: (message: string, description?: string) => {
+      super.logger().error(message, description);
+      this.#dispatchNext({
+        type: MessageLogType.ERROR,
+        title: message,
+        description,
+      });
+    },
+    warn: (message: string, description?: string) => {
+      super.logger().warn(message, description);
+      this.#dispatchNext({
```

Comment:
following latest comment, this means that descoper will get a next, without `state`